doc/examples/segmentation/plot_rolling_ball.py

```diff
@@ -0,0 +1,144 @@
+"""
+=======================
+Background Subtraction using the Rolling Ball Method
+=======================
```

Comment:
Same here, as you can see from the CI build:
https://dev.azure.com/scikit-image/scikit-image/_build/results?buildId=3852&view=logs&j=31eb6c88-23c5-50ae-eda4-a7f72508e1eb&t=f7291549-81f8-5748-7064-2be81963872c

Reply:
fixed - I didn't realize I had access to the CI tests online. That's convenient; thank's for referencing it.

Reply:
Yes, down below this very page, you can click the 'Details' links facing the various "continuous-integration.scikit-image ..." build descriptions. As of 8392d46, you can see there is an error with the reStructuredText reference: https://dev.azure.com/scikit-image/scikit-image/_build/results?buildId=3854&view=logs&j=31eb6c88-23c5-50ae-eda4-a7f72508e1eb&t=f7291549-81f8-5748-7064-2be81963872c